docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -1175,7 +1175,36 @@ UDP
 
   UPSTREAM_PEER_CERT_V_END can be customized using a `format string <https://en.cppreference.com/w/cpp/io/manip/put_time>`_.
   See :ref:`START_TIME <config_access_log_format_start_time>` for additional format specifiers and examples.
-
+%UPSTREAM_PEER_URI_SAN%
```

Comment:
Please re-add the blank line above, and add a blank line between the different operators.

Reply:
Done!